Lib/test/test_xml_etree.py

```diff
@@ -115,6 +121,143 @@ def test_all(self):
         support.check__all__(self, ET, names, blacklist=("HTML_EMPTY",))
 
 
+class ElementTree_Element_UnitTest(unittest.TestCase):
+
+    def test___init__(self):
+        tag = "foo"
+        attrib = { "zix": "wyp" }
+
+        element_foo = ET.Element(tag, attrib)
+
+        with self.subTest("traits of an element"):
+            self.assertIsInstance(element_foo, ET.Element)
+            self.assertIn("tag", dir(element_foo))
+            self.assertIn("attrib", dir(element_foo))
+            self.assertIn("text", dir(element_foo))
+            self.assertIn("tail", dir(element_foo))
+
+        with self.subTest("string attributes have expected values"):
+            self.assertEqual(element_foo.tag, tag)
+            self.assertIsNone(element_foo.text)
```

Comment:
This may be an implementation detail.

Reply:
Shouldn't that be definitively decided one way or the other?

Reply:
Actually it already is tested in test_set_attribute.

Reply:
It's not an implementation detail and there is a general difference between "no text" (i.e. None) and "empty text" (i.e. the empty string), which can be used for formatting.

Agreed that we should avoid duplicating tests, though. Preferably, there should be a single "obvious" place where a feature or property is tested.